### PR TITLE
Added matrices multiplication & gl uniform&attribute location function 

### PR DESCRIPTION
**Pull request by prooxeydev**
This PR adds 2 functions in gl module, which can be used to get uniform locations/attribute locations from the shader, 1 which sets the data at a uniform location. I added also a mult function in the glm module. The function multiplies 2 matrices, tests are working, but they need to improvment.

### CODE REVIEW

**Comment by prooxeydev** [comment]
Added table of contents to docs.md, tried to make an other fork but idk it hasn't worked :/

File: doc/docs.md [hunk 1]
```diff
@@ -16,14 +16,58 @@ you can do in V.
...
 
-<table>
-	<tr>
-		<td><a href='#hello-world'>1. Hello world</a></td>
-		<td><a href='#comments'>2. Comments</a></td>
-		<td><a href='#functions'>3. Functions</a></td>
-		<td><a href='#'>TODO</a></td>
-	</tr>
-</table>
+<ol>
```
Comment (medvednikov) [inline doc/docs.md:19]: I created a table, so that it wouldn't take so much vertical space

File: doc/docs.md [hunk 2]
```diff
@@ -1586,10 +1630,10 @@ Let's create a simple program `test.cpp` first:
 #include <iostream>
 
 int main() {
-        std::vector<std::string> s;
```
Comment (medvednikov) [inline doc/docs.md:1589]: what is this? this should not be changed.

File: doc/docs.md (hunk 1 above)
Comment (prooxeydev) [reply doc/docs.md:19]: i can edit it but ol have automatic numbers on left side

Comment (prooxeydev) [reply doc/docs.md:19]: but i will change it

File: doc/docs.md (hunk 2 above)
Comment (prooxeydev) [reply doc/docs.md:1589]: my fault with replacing things, i will change it back to table and this error too

File: doc/docs.md (hunk 1 above)
Comment (prooxeydev) [reply doc/docs.md]: i fixed it but with a table it isn't look nice

**Pull request merged**